packages/theme/src/cli/utilities/asset-path.ts

```diff
@@ -0,0 +1,9 @@
+import {dirname, joinPath} from '@shopify/cli-kit/node/path'
+import {createRequire} from 'node:module'
+
+const require = createRequire(import.meta.url)
+
+export async function resolveAssetPath(...subpaths: string[]) {
+  const cliRootPath = dirname(require.resolve('@shopify/cli/package.json'))
+  return joinPath(cliRootPath, 'dist', 'assets', ...subpaths)
+}
```

Comment:
Maybe this could be moved to cli-kit eventually? I see `@shopify/app` uses an [ESBuild plugin](https://github.com/Shopify/cli/blob/e2b829d0ef1f292e10fc415a12cd5567683972df/bin/bundling/esbuild-plugin-graphiql-imports.js#L13) to find the assets but perhaps this is simpler? @isaacroldan